src/components/__tests__/Bullet.ts

```diff
@@ -128,9 +128,10 @@ describe('render', () => {
   })
 })
 
-// TODO: findSubthoughts is broken after LayoutTree
-describe.skip('expansion', () => {
+describe('expansion', () => {
   it('tapping an expanded cursor bullet should collapse the thought by moving the cursor up', async () => {
+    const unrankedPath = ['a', 'b']
```

Comment:
Prefer inlining this value. `['a', 'b']` is a short, readable value, so in tests it's better not to abstract it into a variable and force the user to move their eyes back-and-forth when reading through the test case.

This differs from the DRY approach that is taken on the codebase itself, where we want to avoid duplication and tightly couple dependent values. Tests on the other hand benefit from being as concrete as possible.